react/__tests__/Slider.test.tsx

```diff
@@ -42,6 +42,7 @@ describe('Basic rendering', () => {
     const { getByTestId, getByLabelText } = render(
       <Slider
         centerMode="disabled"
+        centerModeSlidesGap={undefined}
```

Comment:
You can simply not pass this prop and its value will be `undefined`.

Reply:
Without it it trows error that property is missing.

Reply:
Oh! That's because the type is wrong then :)
This should be an optional prop

Reply:
Ok, i've removed them. Sorry about that.